### ISSUE TITLE
instruction of route

### ISSUE THREAD

**Issue by liodali** [liodali/osm_flutter#116]
when drawRoad we can get instruction of route
in this issue i will add get instruction of each route with information that already provided duration & distance

**Comment by jesussmile** [comment]
That would be great!  Is it like the list that shows all turns and distance when we select two points?

**Comment by liodali** [comment]
yeah something like that but only problem that i will face that information is in eng 
so i need to find solution how i can make it multi language

**Comment by liodali** [comment]
but that should application take care of it not the plugin 
for first solution maybe i will take of it and i will support en,fr after that i will find another solution
